_posts/2016-08-25-hybox-names.markdown

```diff
@@ -0,0 +1,38 @@
+# What’s in a name? The many facets of Hydra-in-a-Box
+
+Sometimes you need to work on something for a while before you figure out what its name should be. We’re at that stage now. The official name for the project, the one that went in on the grant proposal, describes the overarching objectives of the grant, but it’s anything but pithy: *Fostering a New National Library Network through a Community­-Based, Connected Repository System*. So what should we call what we’re making? (No really--we need your ideas. Read on!)
+
+As we’ve moved through the first year of the project, it has become clear that our effort has many facets, and choosing (and using) the right names has helped differentiate the various strands. We have identified at least five distinct components for our effort, each with its own identity and branding considerations.
+
+1. **the project.** We’ve settled on using **Hydra-in-a-Box** or **HyBox** (for shorthand, and used with some reservation) for the project. The project URL is currently [hydrainabox.org](http://hydrainabox.org), and resolves to [http://hydrainabox.projecthydra.org/](http://hydrainabox.projecthydra.org/). While not all of the work revolves around Hydra (there are significant tie-ins specific to DPLA hubs and DPLA metadata--see below), this name captures one of the essential goals of the effort: to produce a turnkey repository product leveraging, and embraced by, the Hydra community. 
+
+2. **code components.** Hydra’s technical architecture has many components and layers, and an explicit Hydra-in-a-Box goal is to contribute to the overall architecture and code ecosystem. Note that a site may choose to run these HyBox components but *not *run HyBox in its turnkey form, just as many sites now say they run "sufia-based repositories." Under the theory that a rising tide raises all ships, the project team’s focus has been to locate functionality in the right places within the Hydra stack, and *not *to create a product-specific code base where we can avoid it. Accordingly, we are **not** using HyBox or Hydra-in-a-Box as the name of code components, in order to encourage re-use in other projects, broad adoption, and code contribution & maintenance from other Hydra efforts.
```

Comment:
I think "*not *run" needs to change to "*not* run", else the italics go on and on.